resources/qml/Preferences/GeneralPage.qml

```diff
@@ -627,14 +627,25 @@ UM.PreferencesPage
             {
                 width: childrenRect.width
                 height: childrenRect.height
-                text: catalog.i18nc("@info:tooltip", "Should a prefix based on the printer name be added to the print job name automatically?")
+                text: catalog.i18nc("@info:tooltip. Note variable names themselves (ie. machine_name_short, project_name) should not be translated", "Variables: machine_name_short, machine_name, project_name")
 
-                CheckBox
+                Column
                 {
-                    id: prefixJobNameCheckbox
-                    text: catalog.i18nc("@option:check", "Add machine prefix to job name")
-                    checked: boolCheck(UM.Preferences.getValue("cura/jobname_prefix"))
-                    onCheckedChanged: UM.Preferences.setValue("cura/jobname_prefix", checked)
+                    spacing: 4 * screenScaleFactor
+
+                    Label
+                    {
+                        id: jobNameTemplateLabel
+                        text: catalog.i18nc("@label","Print job template:")
+                    }
+
+                    TextField
+                    {
+                        id: jobNameTemplateTextField
+                        width: 200 * screenScaleFactor
```

Comment:
The width could be bigger to fit in the full initialy template (just a suggestion).